.github/workflows/validate_changed_files.sh

```diff
@@ -7,10 +7,8 @@ VALID_FILE_PATHS=('pkg/plugin/sdk/v1alpha2') # Array of filepaths that are allow
 
 # by default the only local branch will be pull/PR#/merge
 # fetch only the latest commit from the 2 branches in question to avoid fetching the entire repo which could be costly
-git fetch --depth 1 origin "$1"
-git fetch --depth 1 origin "$2"
 
-git diff "origin/$1..origin/$2" --name-only | while read -r file; do
+for file in "$@"; do
 
```

Comment:
fwiw if I put `echo "file: $file"` here it does correctly loop over each file in the commit

Reply:
this is the same as the PR but with the echo: https://github.com/aspect-build/aspect-cli/runs/5057836349